monix-eval/jvm/src/test/scala/monix/eval/TaskLocalJVMSuite.scala

```diff
@@ -26,6 +26,7 @@ import monix.execution.exceptions.DummyException
 import monix.execution.{ExecutionModel, Scheduler}
 import monix.execution.misc.Local
 import monix.execution.schedulers.TracingScheduler
+import monix.execution.cancelableFutureCatsInstances
```

Comment:
So, Scala 3 dropped adding object prefixes to implicit scope, so the trick I proposed quite some time ago with making these options work out of the box is broken. I wonder if we can/should adapt something like technique from [here](https://blog.7mind.io/no-more-orphans.html) or simply bite the bullet and drop the "monix-execution doesn't depend on cats directly" policy.

Reply:
I don't know what to do next. I don't like the "no more orphans" solution, as it looks like something that can break in the future. I also noticed that Scala 3 started to give advice on imports, not sure how reliable it is, but it looks good. The Scala 3 compiler actually suggested that import in the error report.

Another possibility would be to have another sub-project called `monix-execution-instances` (also a dependency of `monix-catnap`) and have people do ...

```scala
import monix.execution.instances._
```

And hopefully the Scala 3 compiler will suggest it.

Reply:
What I mean is, if we want an optional dependency on Cats, and if the Scala 3 compiler (aka the future) can suggest the import, then maybe such orphans are fine.

Reply:
Yeah I see what you mean. I'd put those into catnap to not breed submodules.